chapters/usage-profile.md

```diff
@@ -0,0 +1,244 @@
+
+### 31.3.3 Examples
+
+EXAMPLE 1 Tag: `UsageScopeSPDXID:`
+
+```text
+UsageScopeSPDXID: SPDXRef-1
+```
+
+EXAMPLE 2 RDF: The URI for the element will follow the form:
+
+```text
+[SPDX document namespace]#[SPDX identifier]
+```
+
+See [6.5](document-creation-information.md#6.5) for the definition of the SPDX document namespace and [6.3](document-creation-information.md#6.3) for the definition of the SPDX identifier
+
+Using `xml:base`:
+
+```text
+<rdf:RDF xml:base="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B">
+    ...
+    <UsageScope rdf:about="#SPDXRef-1">
+    ...
+    </UsageScope>
+</rdf:RDF>
+```
+
+Using document URI:
+
+```text
+<UsageScope rdf:about="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B#SPDXRef-1">
+    ...
+</UsageScope>
+```
+
+## 31.4 Package Release Date field <a name="31.4"></a>
+
+### 31.4.1 Description
+
+This field provides a place for recording package release date
+
+Table 31.4 — Metadata for the package release date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.4.2 Intent
+
+Description of the package release date for strict identification of the prerequisite assumptions of usage scope by the package supplyer.
+
+### 31.4.3 Examples
+
+EXAMPLE 1 Tag: `PackageReleaseDate:`
+
+```text
+PackageReleaseDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:packageReleseDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <packageReleaseDate> 2010-01-29T18:30:22Z </packageReleaseDate>
+</UsageScope>
+```
+
+## 31.5 Package Built Date field <a name="31.5"></a>
+
+### 31.5.1 Description
+
+This field provides a place for recording built date of the package 
+
+Table 31.5 — Metadata for the package built date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.5.2 Intent
+
+Description of the built date of the package for strict identification of the prerequisite assumptions of usage scope by the package supplyer.
+It's recoreded from build system tools or date field of the package file.
```

Comment:
Note spelling of supplier.

I'm not sure we want to add the "how" in the intent.  Could remove the last line or change it to more of a suggestion or "should".  ie: `It should recorded from build system tools or date field of the package file.`

Also - note spelling of recorded